Debug Installation and CakePHP 2.10.x/PHP 7 compatibility

Nyrtub opened the pull request:
Debug Installation plugin
Compatibility with CakePHP 2.10.x/PHP 7
More informations in Administration > Informations

Nyrtub committed:
Debug Installation and CakePHP 2.10.x/PHP 7 compatibility

Nyrtub committed:
.gitignore & core.php update

Nyrtub committed:
Unexpected field 'User.notified_project_ids' in POST data, Missing field 'User.notified_project_ids][' in POST data

Error: The requested address '/my/account' was not found on this server.

Nyrtub committed:
Return correct formated title

yandod commented:
very impressive change. Thanks for this.
but couldnt review quick because of the size.

Nyrtub committed:
Layout

Nyrtub committed:
Error isUnique($fields, array())

Nyrtub committed:
Some Links target to blank

Nyrtub committed:
Replace isUnique($field, $data) by checkUnique($field, array())

Nyrtub committed:
Helper::Form->create(..., array('action' =>...)) deprecated

Nyrtub committed:
Clean and review fra locale

Nyrtub committed:
ProjectsController::Activity() :
Model Journal.php:83 => JournalDetail.* not loaded
Model Journal.php:170 => $detail['prop_key'] does not exist

Nyrtub committed:
Notice (8): Only variables should be assigned by reference [APP/Model/Issue.php, line 146]

Nyrtub committed:
Deprecated (16384): Using key `action` is deprecated, use `url` directly instead. [CORE/Cake/View/Helper/FormHelper.php, line 383]

Nyrtub committed:
Update RequestHandler detection

Nyrtub committed:
Projects::Add_issue_category error with POST and AJAX

Nyrtub committed:
Problem with french caracters

Nyrtub committed:
Update relations management between issues

Nyrtub committed:
Add Link Break for Issue Relations

Nyrtub committed:
Fixe filters bugs display and translations

Nyrtub committed:
Fixe some display

Nyrtub committed:
Fixe email sending troubles if settings are not correct

Nyrtub committed:
Localize toolbar and tooltips

Nyrtub committed:
Update Form

Nyrtub committed:
Fixe issue if application is used by only one user who don't want receive mail notifications

Nyrtub committed:
Fixe issue with archived project
Display confirm before delete

Nyrtub committed:
Fixe some display problems, bugs and clean code

Nyrtub committed:
Translation

Nyrtub committed:
Bug Fixe if only one user

Nyrtub committed:
Enumeration Translation

Nyrtub committed:
Translation

yandod commented:
Thanks for this work. but I am very afraid to say, this is too huge and I can not review this.
if we proceed this, we need to stay on branch and review by chunk by chunk.

Nyrtub commented:
I understand.
Regards

Nyrtub committed:
Undefined variable: project_tree if no public project

rkjamvi commented:
Is it ready for PHP7 now?
